### PR TITLE
Update README.md for windows vscode

### PR DESCRIPTION
**Pull request by dsisnero**
Add information on windows needing port to connect

Thanks for your Pull Request 🎉 

**Please follow these instructions to help us review it more efficiently:**

- Add references of related issues/PRs in the description if available.
- If you're updating the readme file, make sure you followed [the instruction here](https://github.com/ruby/debug/blob/master/CONTRIBUTING.md#to-update-readme).

## Description
Describe your changes:

### CODE REVIEW

**Comment by ono-max** [comment]
Thanks, could you follow the following instruction?
https://github.com/ruby/debug/blob/master/CONTRIBUTING.md#to-update-readme

File: README.md [hunk 1]
```diff
@@ -362,6 +362,21 @@ If you don't run a debuggee Ruby process on VSCode, you can attach with VSCode l
 
 `rdbg --open=vscode` opens the debug port and tries to invoke the VSCode (`code` command).
 
+On windows need to use port
+`rdbg --open=vscode port 1234`
```
Comment (ko1) [inline README.md:366]: `--port` instead of `port`?

Comment (ko1) [reply README.md:366]:
```suggestion
`rdbg --open=vscode --port 1234`
```

File: README.md [hunk 2]
```diff
@@ -362,6 +362,21 @@ If you don't run a debuggee Ruby process on VSCode, you can attach with VSCode l
...
+
+and need to add port to launch.json file
+```
+   {
+      "name": "Listen for rdebug-ide",
+      "type": "Ruby",
```
Comment (ko1) [inline README.md:372]: does it work with rdbg?

**Comment by ko1** [comment]
Ah, `open` command or `--open=vscode` doesn't work now because `launch.json` is auto-generated.
We need to modify the code, and the document update is not needed.
I'll left this issue as pending and try it later (maybe next and next version).

**Pull request closed without merging**